core-bundle/contao/templates/twig/content_element/swiper.html.twig

```diff
@@ -0,0 +1,78 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+{% use "@Contao/component/_stylesheet.html.twig" %}
+
+{% set slider_attributes = attrs(slider_attributes|default)
+    .addClass('swiper')
+    .set('data-delay', delay)
+    .set('data-speed', speed)
+    .set('data-offset', offset)
+    .set('data-loop', loop)
+%}
+
+{% block content %}
+    <div{{ slider_attributes }}>
+        <div{{ attrs(slider_wrapper|default).addClass('swiper-wrapper') }}>
+            {% block slides %}
+                {% for element in nested_fragments %}
+                    {% block slide %}
+                        <div{{ attrs(element_slide|default).addClass('swiper-slide') }}>
+                            {% block slide_inner %}
+                                {{ content_element(element) }}
+                            {% endblock %}
+                        </div>
+                    {% endblock %}
+                {% endfor %}
+            {% endblock %}
+        </div>
+        {% block controls %}
+            <button{{ attrs(slide_button_prev|default).set('type', 'button').addClass('swiper-button-prev') }}></button>
+            <button{{ attrs(slide_button_next|default).set('type', 'button').addClass('swiper-button-next') }}></button>
+            <div{{ attrs(pagination|default).addClass('swiper-pagination') }}></div>
```

Comment:
```suggestion
            <div{{ attrs(pagination_attributes|default).addClass('swiper-pagination') }}></div>
```
I’d name this one differently I think to prevent confusion.
Feel free to ignore.

Reply:
> What is the benefit of the `_attributes` suffix?

It’s a hint that it is an `HtmlAttributes` object. We use `_attributes` or `_attrs` pretty often as a suffix in other templates or components AFAIR. I’d favor using the suffix for all four variables, yes.

Reply:
Changed in 3901c90e9a6264102130c830aa70c0b099cef322.